addons/binding/org.openhab.binding.enocean/README.md

```diff
@@ -0,0 +1,128 @@
+# EnOcean Binding
+
+The EnOcean binding connects OpenHab to the EnOcean ecosystem.
+
+The binding uses an EnOcean gateway to retrieve sensor data and control actuators. For _bidirectional_ actuators it is even possible to update the OpenHab item state if the actuator gets modified outside of OpenHab.
+This binding has been developed on an USB300 gateway and was also tested with an EnOceanPi. As this binding implements a full EnOcean stack, we have full control over these gateways. This binding can enable the repeater function (level 1 or 2) of these gateways and retrieve detailed information about them.   
+
+## Concepts/Configuration
+
+First of all you have to configure an EnOcean Transceiver (Gateway). This device has to be added manually to OpenHab and is represented by an _EnOcean bridge_. You just have to set the right serial port. If everything is running fine you should see the _base id_ of your gateway in the properties of your bridge.
+
+EnOcean messages are mainly send as broadcast messages without an explicit receiver address. However each message contains a unique sender address (EnOcean Id) to determine from which device this message was sent. To receive messages from an EnOcean device you have to determine its EnOcean Id and add an appropriate thing to OpenHab. **The thing Id has to be set to the EnOcean Id**. If the device is an actuator which you want to control, you have to generate an unique sender id and announce it to the actuator (_teach-in_). This sender id is made up of the base id of the EnOcean gateway and a number between 1 and 127. This number can be set manually or determined by the binding.
+
+## Supported Things
+
+This binding is developed on and tested with the following things
+
+ * USB300 and EnOceanPi gateways as OpenHab bridges
+ * The following Eltako actuators:
+    * FSR14 (light switch)
+    * FSB14 (rollershutter)
+    * FUD14 (dimmer)
+    * FSSA-230V (smart plug)
+ * The following Opus actuators:
+    * GN-A-R12V-SR-4 (light)
+    * GN-A-R12V-MF-2 (light)
+    * GN-A-R12V-LZ/UD (dimmer)
+    * GN-A-R12V-JRG-2 (rollershutter)
+    * GN-A-U230V-JRG (rollershutter)
+    * OPUS-FUNK PLUS Jalousieaktor 1fach UP (rollershutter)
+    * OPUS-Funk PLUS Steckdosenleiste (smart multiple socket)
+ * NodOn Smart Plug (ASP-2-1-10), In Wall Switch (SIN-2-2-00, SIN-2-1-0x), Permundo PSC234 (smart plug with metering)
+ * Thermokon SR04 room control
+ * Hoppe SecuSignal window handles
+ * Rocker switches (NodOn, Eltako FT55 etc)
+
+However because of the standardized EnOcean protocol it is more important which EEP this binding supports: F6-02, F6-10, D5-00, A5-10, A5-38 (switching and dimming), A5-37 (use generic things). Hence if your device supports one of these EEPs the chances are good that your device is also supported by this binding.
```

Comment:
You should provide a table with the exact thing type ids and their description.

Reply:
fixed